libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/LaneIconProcessor.kt

```diff
@@ -75,6 +83,17 @@ internal object LaneIconProcessor {
             laneIndicator.directions.size == 1 && laneIndicator.directions[0] == SHARP_RIGHT -> {
                 laneIcon[LaneTurns.LANE_SHARP_RIGHT]
             }
+            laneIndicator.directions.size > 1 &&
+                laneIndicator.directions.contains(LEFT) &&
+                laneIndicator.directions.contains(RIGHT) -> {
```

Comment:
It looks like this codebase hasn’t been updated to handle the very common left/straight/right combination that was added in mapbox/navigation-ui-resources#21. This heuristic would display a left/right icon even if the active direction is straight. For now, would it be possible to display just a straight arrow in this case? The same special case would also be useful for the slight left/straight/slight right combination, which is less common.

Reply:
@1ec5 Added in this [commit](https://github.com/mapbox/mapbox-navigation-android/pull/4815/commits/ce6d3e47f9e2d91470b0e313457f75845dc631d8)